perception/autoware_tensorrt_bevdet/package.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>autoware_tensorrt_bevdet</name>
+  <version>0.0.1</version>
+  <description>tensorrt library implementation for bevdet</description>
+
+  <author email="cynthia.liu@autocore.ai">Cynthia</author>
+  <maintainer email="cynthia.liu@autocore.ai">Cynthia</maintainer>
+  <maintainer email="yliuhb@connect.ust.hk">Yuxuan Liu</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <buildtool_depend>autoware_cmake</buildtool_depend>
+  <buildtool_depend>cudnn_cmake_module</buildtool_depend>
+  <buildtool_depend>tensorrt_cmake_module</buildtool_depend>
+
+  <buildtool_export_depend>cudnn_cmake_module</buildtool_export_depend>
+  <buildtool_export_depend>tensorrt_cmake_module</buildtool_export_depend>
+
+  <depend>autoware_perception_msgs</depend>
+  <depend>cv_bridge</depend>
+  <depend>geometry_msgs</depend>
+  <depend>libopencv-dev</depend>
+  <depend>pcl_conversions</depend>
+  <depend>pcl_ros</depend>
+  <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+  <depend>sensor_msgs</depend>
+  <depend>tensorrt_common</depend>
+  <depend>tf2_geometry_msgs</depend>
```

Comment:
The dependency on `yaml-cpp` should be reviewed. Ensure that this dependency is strictly necessary. If  `yaml-cpp` is used only in a specific part of the code, consider making it a conditional dependency to reduce unnecessary dependencies for users who don't require that specific functionality. This improves build speed and reduces the overall size of the project.